content/ember/v5/deprecate-array-prototype-extensions.md

```diff
@@ -0,0 +1,1265 @@
+---
+id: deprecate-array-prototype-extensions
+title: "Deprecate array prototype extensions"
+until: '6.0.0'
+since: '5.0.0'
```

Comment:
Should we update this to the current one on `main`, assuming we get the deprecation implemented in that version?

```suggestion
since: '5.10.0'
```